File: backend/src/main/java/nextpos/app/nextpos/model/enums/PriceListType.java
package nextpos.app.nextpos.model.enums;

public enum PriceListType {
    DEFAULT, // Standard base price
    WHOLESALE, // Wholesale discounted price
    RETAIL, // Explicit retail pricing (if separate from default)
    SEASONAL, // Seasonal or festival price
    PROMOTIONAL, // Limited-time promotions
    CLEARANCE, // Stock clearance pricing
    CUSTOM // Catch-all for company-specific lists
}

File: frontend/src/enums/PriceListType.ts
export enum PriceListType {
  DEFAULT = "DEFAULT",
  WHOLESALE = "WHOLESALE",
  RETAIL = "RETAIL",
  SEASONAL = "SEASONAL",
  PROMOTIONAL = "PROMOTIONAL",
  CLEARANCE = "CLEARANCE",
  CUSTOM = "CUSTOM",
}

export const PriceListTypeLabels: Record<PriceListType, string> = {
  [PriceListType.DEFAULT]: "Default",
  [PriceListType.WHOLESALE]: "Wholesale",
  [PriceListType.RETAIL]: "Retail",
  [PriceListType.SEASONAL]: "Seasonal",
  [PriceListType.PROMOTIONAL]: "Promotional",
  [PriceListType.CLEARANCE]: "Clearance",
  [PriceListType.CUSTOM]: "Custom",
};
